cfg/rippled-example.cfg

```diff
@@ -758,7 +758,9 @@
 #   When searching for paths, the default search aggressiveness. This can take
 #   exponentially more resources as the size is increased.
 #
-#   The default is: 7
+#   The recommended value to support advanced pathfinding is: 7
```

Comment:
why though?

Reply:
I'm not aware of anything special about `7` other than the fact that it worked well for many years, and is a balance between being unable to find an acceptable path vs. resource consumption. @ximinez @JoelKatz any thoughts to share here?

Reply:
Yep. That's pretty much it. I kind of intentionally left it vague because anybody who doesn't specifically want pathfinding will be just fine with the defaults, and anyone who _does_ probably just wants a reasonable recommendation. If they are more advanced than that, then any "tl;dr" summary in the config file is going to be insufficient - those users should dig into the code or other documentation.